File: codewars/cpp/multiples_of_3_or_5.cpp
// https://www.codewars.com/kata/514b92a657cdc65150000006/train/cpp


#include <bits/stdc++.h>
using namespace std;

int solution(int number) {
    int sum = 0;

    for(int i = 1; i < number; ++i) {
        if(i % 3 == 0 || i % 5 == 0) {
            sum += i;
        } else {
            sum += 0;
        }
    }
    return sum;
}

int main() {
    int number;

    cin >> number;
    cout << solution(number) << endl;

    return 0;
}

File: codewars/java/MultiplesOf3Or5.java
// https://www.codewars.com/kata/514b92a657cdc65150000006/train/java

public class MultiplesOf3Or5 {
	public static void main(String[] args) {
		MultiplesOf3Or5 multiplesOf3Or5 = new MultiplesOf3Or5();
		System.out.println(multiplesOf3Or5.solution(10));
	}

	public int solution(int number) {
		int sum = 0;

		for (int i = 0; i < number; ++i) {
			if (i % 5 == 0 || i % 3 == 0) {
				sum += i;
			}
		}

		return sum;
	}
}
